### ISSUE TITLE
Regularization or Drop-out?

### ISSUE THREAD

**Issue by edu-gp** [ThilinaRajapakse/simpletransformers#209]
Hello, this is a question, not a bug.

I see my text classification model overfitting on a training set, and its performance is far from the test set (and I'm using a validation set to stop training early). Is there a way to add regularization for Transformer models? Does it make sense to do so?

Thank you!

**Comment by ThilinaRajapakse** [comment]
I can't think of any such methods off the top of my head, but I also can't recall needing to use additional regularization or dropout even on tiny datasets. Is the test data significantly different from the training data?

**Comment by kinoute** [comment]
You can try some regularization with the `weight_decay` arg and see how it impacts your training.

**Comment by AdityaSoni19031997** [comment]
Modify the bert_config dropout directly :)
